coding-concepts.js

```diff
@@ -69,5 +68,5 @@ class Learn {
 var learnStudent = new Learn("George")
 // console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Student: George, cohort: "Alpha", year: 2022
+// b) Verify and explain: Because classes are dynamic, it's reusable just like functions. It can take arguments and creates objects with unique data and each object created from the class is independent from each other so it can be reassigned. 
```

Comment:
Awesome!!!  This is a tricky concept to understand and it looks like you got it!